gradle.properties

```diff
@@ -3,6 +3,7 @@ GROUP=io.arrow-kt
 VERSION_NAME=0.10.5-SNAPSHOT
 # Other versions
 KOTLIN_VERSION=1.3.50
+KOTLIN_COROUTINES_VERSION=1.3.3
```

Comment:
Hi @arkivanov !! Thanks for your contribution :tada: Please, I think `KOTLINX_COROUTINES_VERSION` can be used instead. Versions have been ordered now to avoid overlapping. Thanks!!